src/test/java/com/start/code/TestBubbleSort.java

```diff
@@ -0,0 +1,16 @@
+package com.start.code;
+
+import com.start.code.BubbleSort;
+
+import junit.framework.TestCase;
+
+public class TestBubbleSort extends TestCase {
+	
+	public void testBubbleSort() { 
```

Comment:
Rename this test method, to state what is being tested in this particular test.
And add few more tests.